docker-compose.yml

```diff
@@ -19,33 +26,39 @@ services:
             PMA_USER: root
             PMA_PASSWORD: r00tp@ss!
 
-    restore:
+    app:
         build: ./
         volumes:
             - ./application:/data
-#        volumes:
-#            - ./test/world.sql.gz:/tmp/world.sql.gz
-#            - ./test/employees.sql.gz:/tmp/employees.sql.gz
+            - ./test/world.sql.gz:/root/world.sql.gz
+            - ./test/world.sql.sha256.gz:/root/world.sql.sha256.gz
+            - ./test/s3cfg:/root/.s3cfg
         env_file:
             - ./local.env
+            - ./test/cert.env
         environment:
             MYSQL_HOST: db
             MYSQL_USER: root
             MYSQL_PASSWORD: r00tp@ss!
-            DB_NAMES: world #employees
+            S3_BUCKET: s3://world
```

Comment:
The S3_BUCKET should be defined in local.env, so don't override it.

Reply:
Actually, no. The local.env file _can_ have S3_BUCKET but will not necessarily have it. In fact, by including it here, it saves one detail after cloning the repo. (Also, IMHO, local.env should only have sensitive information that can't be committed to the repo.)

Reply:
> The local.env file _can_ have S3_BUCKET but will not necessarily have it.

Right, but the first thing you do after cloning the repo is copying the local.env.dist to local.env and then filling in values. I still think this S3_BUCKET is not necessary here, since I don't have a s3://world bucket and getting `make test` to work requires a bucket that does exist.

Reply:
I think that's the wrong direction. As such, please create a YouTrack issue to improve the documentation on how to actually test this in the README.md file. Because I can't now, cause I don't know how to generate and place the necessary PEM files.